src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests_ListPatterns.cs

```diff
@@ -1450,6 +1450,7 @@ public void M(int[] a)
         _ = a is [>0] and [>=0];
         _ = a is {Length:-1};         // 6
         _ = a is [.., >0] and [<0];   // 7
+        _ = a is [.., >0, _] and [_, <=0, ..];
```

Comment:
This test demonstrate the case where an indexer test would not permit the other with a specific length.
e.g. `[.., >0, _] and [_, <=0, ..]` becomes `length >= 2 && [^1] >0 && (length != 3 && [1]<=0)`